posts/2023-08-30_pharmaverse_story/pharmaverse_story.qmd

```diff
@@ -0,0 +1,103 @@
+---
+title: "The pharmaverse (hi)story"
+author:
+  - name: Nicholas Eugenio
+description: ""
+date: "2023-08-30"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [community]
+# feel free to change the image
+image: "pharmaverse.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2023-08-30_pharmaverse_story"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Pharmaverse: from motivation to present
+
+**Human history and pharmaverse context**
+
+Since the Australian Aboriginal, the earliest peoples recorded to have inhabited the Earth and who have been in Australia for at least 65,000 to 80,000 years ([Encyclopædia Britannica](https://www.britannica.com/topic/Australian-Aboriginal)), human beings live in group.
+Whether to protect yourself, increase your life expectancy or simply share tasks.
+
+For most aspects of life, it doesn't make sense to think, act or work alone for two main reasons:
+
+1.  You will spend more energy and time;
+
+2.  Someone else may be facing (or have faced) the same situation.
+
+The English poet John Donne used to say *"No man is an island entire of itself; every man is a piece of the continent, a part of the main;"*.
+I can't disagree with him.
+And I dare say that Ari Siggaard Knoph (Novo Nordisk), Michael Rimler (GSK), Michael Stackhouse (Atorus), Ross Farrugia (Roche), and Sumesh Kalappurakal (Janssen) can't disagree with him either.
+They are the founders of [pharmaverse](https://pharmaverse.org/), members of its Council and kindly shared their memories of how independent companies, in mid-2020, worked together in the creation of a set of packages developed to support the clinical reporting pipeline.
+
+If you are not familiar with this pipeline, the important thing to know is that, in a nutshell, pharmaceutical companies must follow a bunch of standardized procedures and formats (from Clinical Data Interchange Standards Consortium, [CDISC](https://www.cdisc.org/)) when submitting clinical results to Health Authorities.
+Focus on this: **different** companies seeking the **same** standards for outputs.
```

Comment:
This sentence looks a bit like a note to yourself. 
Maybe write: 
The focus is on this: **different** companies ...